src/actions/pause/mapFrames.js

```diff
@@ -14,11 +14,13 @@ import {
 import assert from "../../utils/assert";
 import { findClosestFunction } from "../../utils/ast";
 
-import type { Frame, ThreadId } from "../../types";
+import type { Frame, OriginalFrame, ThreadId } from "../../types";
 import type { State } from "../../reducers/types";
 import type { ThunkArgs } from "../types";
 
+/* eslint-disable import/no-duplicates */
```

Comment:
I couldn't sort out a way to import `typeof SourceMaps` and `{ isGeneratedId }` in a way that made everything happy. I elected to suppress the linting error since the code style deviation was necessary for typing. Another alternative would be to import the default export and create a type from it but importing "real" code just to then create a type seemed "more bad" to me.

```
import DevToolsSourceMap, {isGeneratedId} from "devtools-source-map";
type SourceMaps = typeof DevToolsSourceMap;
```